PermissionsEx/permissions.yml

```diff
@@ -1113,6 +1113,8 @@ groups:
         permissions:
         - ch.alias.mr.shifty
         - -essentials.protect.damage.fall
+        - worldedit.navigation.thru.command
+        - worldedit.navigation.jumpto.command 
```

Comment:
giving him ender pearls would be fine, but even ender pearls go farther than 2 feet